web/styles/subscriptions.css

```diff
@@ -1405,16 +1354,18 @@ div.settings-radio-input-parent {
         max-width: 95%;
     }
 
-    .two-pane-settings-container .left .list-toggler-container {
-        flex-wrap: wrap;
-    }
-
     #groups_overlay .group_settings_header,
     #subscription_overlay .stream_settings_header {
         flex-wrap: wrap;
     }
 }
 
+@container settings-overlay (width >= $settings_overlay_sidebar_collapse_breakpoint) {
+    .two-pane-settings-container .right {
```

Comment:
I cannot understand why this CSS is needed only for certain widths. I removed this and did not find any change in behavior.

Reply:
I accidentally had `resize_settings_overlay_subheader_for_narrow_screens` firing all the time, without it we need to make sure the `right` takes 100% of its height or else the border looks like this:

<img width="905" alt="image" src="https://github.com/user-attachments/assets/29209a9f-6bff-4c15-b396-0d72493e6044" />

But if it's 100% height on narrow screens, it takes up way too much space because `right` is `absolute` positioned. 

I feel like this is still so messy but I've been having trouble figuring out something cleaner. Would love thoughts/ideas if you have any!